### PR TITLE
added a related link to Specification archive for each interface

### PR DESCRIPTION
**Pull request by chillkang**
## Done
added a related link to Specification archive for each interface

## How to QA
- Go to https://charmhub-io-1516.demos.haus/interfaces/ingress
- Check if "Speficifation folder" links to a GH link 

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-2146

## Screenshots
Screen Shot 2023-02-24 at 2 46 45 PM

### CODE REVIEW

File: static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx [hunk 1]
```diff
@@ -221,6 +221,9 @@ function InterfaceDetails() {
                   &nbsp;&nbsp;Submit a bug
                 </a>
               </p>
+              <p>
```
[the remark this answers is not in the document]
Comment (chillkang) [reply static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx:221]: I included this part :)

**Review by Lukewh — APPROVED** [review]
:+1: LGTM - p.s. the demo lilnk is without the `-v0` at the end now :)

File: static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx [hunk 2]
```diff
@@ -217,6 +217,14 @@ function InterfaceDetails() {
                   &nbsp;&nbsp;Submit a bug
                 </a>
               </p>
+              {!isCommunity && (
+              <p>
+                <a href={`https://github.com/canonical/charm-relation-interfaces/tree/main/interfaces/${interfaceName}${interfaceData?.version ? `/${interfaceData.version}` : ""}`}>
```
Comment (Lukewh) [inline static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx:222]: aaactually, this should not include the `/${interfaceData.version}` at the end of the url, it should go to the folder that contains the version folders

**Review by Lukewh — CHANGES_REQUESTED** [review]

**Comment by Zoospora** [comment]
Nice work!! I would change (if possible)  the specification archive icon for this other called "topic"
image

**Comment by anasereijo** [comment]
LGTM ux +1

File: static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx (hunk 2 above)
Comment (chillkang) [reply static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx]: I removed it too :)

**Comment by chillkang** [comment]
> Nice work!! I would change (if possible) the specification archive icon for this other called "topic" image

**Review by Lukewh — APPROVED** [review]
:+1: LGTM!

**Pull request merged**